plugins/woocommerce/includes/rest-api/Controllers/Version2/class-wc-rest-orders-v2-controller.php

```diff
@@ -210,6 +210,29 @@ protected function get_order_item_data( $item ) {
 
 		// Expand meta_data to include user-friendly values.
 		$formatted_meta_data = $item->get_all_formatted_meta_data( null );
+
+		// Filter out product variations.
+		if ( isset( $product ) && 'true' === $this->request['order_item_display_meta'] ) {
+			$order_item_name   = $data['name'];
+			$data['meta_data'] = array_filter(
+				$data['meta_data'],
+				function( $meta ) use ( $product, $order_item_name ) {
+					$meta->key     = rawurldecode( (string) $meta->key );
+					$meta->value   = rawurldecode( (string) $meta->value );
+					$attribute_key = str_replace( 'attribute_', '', $meta->key );
+					$display_key   = wc_attribute_label( $attribute_key, $product );
+					$display_value = wp_kses_post( $meta->value );
```

Comment:
I could swear I added this same comment in my original review ... but it seems to have disappeared, unless it was marked as resolved and I just don't see that in the history [?]

Anyway, will post a second time just to be sure you see this. What about the following:

```suggestion
					$display_value = wp_kses_post( rawurldecode( (string) $meta->value ) );
```

Reply:
That is, we don't seem to use `$display_key`, which can be removed, and we only form `$attribute_key` to help us form `$display_key`, so that can also be removed; in turn, we do not need `$meta->key` which is only pulled to form the latter.